updatecli/update-cli-ref-docs.yaml

```diff
@@ -1,11 +1,10 @@
 policies:
   - name: Update kwctl CLI reference documentation
-    policy: ghcr.io/jvanz/updatecli/policies/file:0.1.0
+    policy: ghcr.io/jvanz/updatecli/policies/file:0.1.1
```

Comment:
Once this [PR](https://github.com/updatecli/policies/pull/43) is merged, we can update here to use the official policy version